src/Avalonia.Controls/Converters/PlatformKeyGestureConverter.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Text;
+using Avalonia.Data.Converters;
+using Avalonia.Input;
+
+namespace Avalonia.Controls.Converters
+{
+    public class PlatformKeyGestureConverter : IValueConverter
```

Comment:
Wondering if that could be exposed as a static helper since I would like to reuse that logic in our app and I would need to create a new instance of this converter to do so otherwise (I can obviously use a static shared instance but that feels hacky).

Reply:
Done!